core/src/main/java/com/klaytn/caver/kct/kip_wrapper/KIP7Wrapper.java

```diff
@@ -0,0 +1,106 @@
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.math.BigInteger;
+import java.util.Map;
+
+/**
+ * Representing a KIP7Wrapper
+ * 1. This class wraps all static methods of KIP7
+ * 2. This class should be accessed via `caver.kct.kip7`
+ */
+public class KIP7Wrapper {
+    /**
+     * A Caver instance
+     * this should be injected from KCTWrapper(= `caver.kct`)
+     */
+    private Caver caver;
+
+    /**
+     * Creates a KIP7Wrapper instance
+     * @param caver A Caver instance
+     */
+    public KIP7Wrapper(Caver caver) {
+        this.caver = caver;
+    }
+
+    /**
+     * Deploy KIP-7 contract.
+     * It must add deployer's keyring in caver.wallet.
+     * @param deployer A deployer's address.
+     * @param name A KIP-7 contract name.
+     * @param symbol A KIP-7 contract symbol.
+     * @param decimals A KIP-7 contract decimals.
+     * @param initialSupply A KIP-7 contract initial supply.
+     * @return KIP7
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public KIP7 deploy(String deployer, String name, String symbol, int decimals, BigInteger initialSupply) throws NoSuchMethodException, IOException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
+        return KIP7.deploy(this.caver, deployer, name, symbol, decimals, initialSupply, this.caver.getWallet());
+    }
+
+    /**
+     * Deploy KIP-7 contract.
+     * It must add deployer's keyring in caver.wallet.
+     * @param tokenInfo The KIP-7 contract's deploy parameter values
+     * @param deployer A deployer's address
+     * @return KIP7
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public KIP7 deploy(KIP7DeployParams tokenInfo, String deployer) throws NoSuchMethodException, TransactionException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
+        return KIP7.deploy(this.caver, tokenInfo, deployer, this.caver.getWallet());
+    }
+
+    /**
+     * Detects which interface the KIP-7 token contract supports.<p>
+     * Example :
+     * <pre>{@code
+     * Map<String, Boolean> result = KIP7.detectInterface();
+     * result.get(KIP7.INTERFACE_ID_IKIP7);
+     * result.get(KIP7.INTERFACE_ID_IKIP7_BURNABLE);
+     * result.get(KIP7.INTERFACE_ID_IKIP7_METADATA);
+     * result.get(KIP7.INTERFACE_ID_IKIP7_MINTABLE);
+     * result.get(KIP7.INTERFACE_ID_IKIP7_PAUSABLE);
+     * }</pre>
+     *
+     * @param contractAddress
+     * @return
```

Comment:
```suggestion
     * @return Map<String, Boolean>
```
Can you fill return type of those `detectInterface` function?

Reply:
@jimni1222 
Of course.
I'll :) 

Thanks for your feedback.